src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/BetaVersionWarning.java

```diff
@@ -16,14 +18,10 @@
    * @param version Version of the software.
    */
   public BetaVersionWarning(@NotNull Version version) {
-    super("A+",
-        "A+ Courses plugin is under development",
-        "You are using version " + version + " of A+ Courses plugin, "
-            + "which is a pre-release version of the plugin and still under development. "
-            + LineSeparator.getSystemLineSeparator().getSeparatorString()
-            + "Some features of this plugin are still probably missing, "
-            + "and the plugin is not yet tested thoroughly. "
-            + "Use this plugin with caution and on your own risk!",
+    super(
+        getText("general.aPlus"),
+        getText("notification.BetaVersionWarning.title"),
+        getAndReplaceText("notification.BetaVersionWarning.content", version),
```

Comment:
Can't we remove this notification since we removed the version checking activity?

Reply:
I think I did it already (maybe in another branch, but will re-check, thanks)